Userland/Applications/Welcome/WelcomeWindow.gml

```diff
@@ -25,7 +25,7 @@
                 min_width: 340
                 min_height: 170
                 fill_with_background_color: true
-                background_role: "Base"
+                background_role: @ColorRole::Base
```

Comment:
This change (and the similar changes below) is incorrect. The problem is the GML compiler doesn't know that this property accepts an enum. If you add these properties to the HashMap here, and then revert them to the previous quoted strings, I think that should fix it:
https://github.com/SerenityOS/serenity/blob/5e1499d10450bb093035eadeb8e16cf5151ab047/Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp#L173-L183

Reply:
Done!